darts/models/forecasting/forecasting_model.py

```diff
@@ -907,6 +907,7 @@ def historical_forecasts(
     def backtest(
         self,
         series: Union[TimeSeries, Sequence[TimeSeries]],
+        historical_forecasts: Union[TimeSeries, Sequence[TimeSeries]] = None,
```

Comment:
Could you add the parameter definition to the docstring?

Reply:
On it